stat/distuv/distribution_test.go

```diff
@@ -261,7 +265,7 @@ func testRandLogProbContinuous(t *testing.T, i int, min float64, x []float64, f
 			return math.Exp(f.LogProb(x))
 		}
 		// Integrate the PDF to find the CDF
-		estCDF := quad.Fixed(prob, min, pt, 1000, nil, 0)
+		estCDF := quad.Fixed(prob, min, pt, 10000, nil, 0)
```

Comment:
Why was this necessary?

Reply:
PDFs with singularities (like Gamma for alpha < 1).

Reply:
Thanks.